ownCloud/UIKit Extensions/String+Extension.swift

```diff
@@ -23,4 +23,8 @@ extension String {
     var localized: String {
         return NSLocalizedString(self, comment: "")
     }
+
+	func fileExtension() -> String? {
+		return NSURL(fileURLWithPath: self).pathExtension
```

Comment:
Maybe I can change the location of this method to `OCItem+Extension.swift` renaming it to `item extension` to be able to reuse it in the future if it's needed instead of coding it again. Or even create a new property in `OCItem+Extension.swift` like:
```Swift
var extension: String {
 return (name as NSString).pathExtension
}
```
What do you think?

Reply:
`extension`, of course, would be best, but I'm afraid of colliding with what Apple might provide in the future. What do you think about `fileSuffix`? That's recognizable for us, but not a reserved Swift keyboard and not what Apple could likely choose in the future (`pathExtension`).